src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/AixPPC64Linker.java

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2020, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023 SAP SE. All rights reserved.
```

Comment:
This file no longer needs any modification relative to upstream. `ValueLayout.JAVA_DOUBLE` already has 4-byte alignment on AIX so the modifications here don't accomplish anything.

Reply:
Restored the upstream version.